lale/operators.py

```diff
@@ -423,7 +423,7 @@ def pretty_print(
         show_imports: bool = True,
         combinators: bool = True,
         assign_nested: bool = True,
-        customize_schema: bool = False,
+        include_customize_schema: bool = False,
```

Comment:
Same comment as in pretty_print.py: this breaks backward compatibility for a popular function.

Reply:
that is a reasonable objection; reverting this change for backward compatibility reasons